src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -258,8 +256,8 @@ export default class AirportModel {
             const lines = map;
 
             _forEach(lines, (line) => {
-                const start = PositionModel.calculatePosition([line[0], line[1]], this.position, this.magnetic_north);
-                const end = PositionModel.calculatePosition([line[2], line[3]], this.position, this.magnetic_north);
+                const start = PositionModel.calculateRelativePosition([line[0], line[1]], this.position.relativePosition, this.magnetic_north);
+                const end = PositionModel.calculateRelativePosition([line[2], line[3]], this.position.relativePosition, this.magnetic_north);
```

Comment:
I don't like these inline arrays at all, its hard to read and tough to tell whats going on.  Could you add a `TODO` comment in this method to move those inlines to vars first? 

example:  `startPosition = [line[0], line[1]];`